lib/licensed/sources/pipenv.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Licensed
+  module Sources
+    class Pipenv < Source
+      def enabled?
+        Licensed::Shell.tool_available?("pipenv") && File.exist?(@config.pwd.join("Pipfile.lock"))
```

Comment:
for clarification since I don't have much experience in this ecosystem - does this have potential overlap with the `pip` source?  Is there likely to be both a `Pipfile.lock` and `requirements.txt` in the same repo?

Reply:
Also, in the pip source the `pip` executable that's used is found based on configuration settings.  I'm assuming that's not needed here because `pipenv` knows how to find the right executable, is that correct?

Reply:
Yes, `pipenv` will run the correct `pip` (the one that's in the virtual environment managed by `pipenv`).

For the first comment (about both `Pipfile.lock` and `requirements.txt` coexisting) -- theoretically there is such possibility to have both, but it does not make much sense, `requirements.txt` is used in non-Pipenv situations (like no venv at all, global packages, manual `virtualenv`) while `Pipfile.lock` is exclusive to `pipenv`. If both are present user would probably like to select one or the other source for `licensed`, otherwise, he/she will get output from both (which does not do any harm in my opinion anyway).